activities/MazeWeb.activity/js/activity.js

```diff
@@ -53,6 +53,24 @@ define(["sugar-web/activity/activity","tween","rAF","activity/directions","sugar
             }
         });
 
+        document.getElementById("fullscreen-button").addEventListener('click', function() {
+            document.getElementById("main-toolbar").style.opacity = 0;
+            console.log("1");
```

Comment:
Don't forget to remove debugging messages

Reply:
Sorry! Fixed now.